tests/test-cases/Group1-Docker-Commands/1-27-Docker-Login.robot

```diff
@@ -34,5 +41,20 @@ Docker login and pull from docker.io
     Should Be Equal As Integers  ${rc}  0
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} logout
     Should Be Equal As Integers  ${rc}  0
-	
-	
+
+    Install Harbor To Test Server  name=19-4-harbor  protocol=https
+    Log To Console  Harbor installer completed successfully...
+
+    Set Environment Variable  HARBOR_IP  %{HARBOR-IP}
+    
+    Install Harbor Self Signed Cert
+    Install VIC Appliance To Test Server  vol=default --registry-ca=/etc/docker/certs.d/%{HARBOR-IP}/ca.crt  certs=${false}
+    
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} login --username=%{TEST_USERNAME} --password=%{TEST_PASSWORD}
+    Should Contain  ${output}  Login Succeeded
+    Should Be Equal As Integers  ${rc}  0
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} login --username=%{TEST_USERNAME} --password=incorrectPassword
+    Should Contain  ${output}  incorrect username or password
+    Should Be Equal As Integers  ${rc}  1
+
```

Comment:
A test for truncated password (in addition to the plus 1 test above) and equivalent username tests please.